docs/en/OLED_Display.md

```diff
@@ -0,0 +1,199 @@
+# Oled Display
+Extension to use for your build with OLED Display
```

Comment:
```suggestion
Extension to use for your build with OLED Display.
```
Sentences end with a full stop. Please check the rest of this document for consistency.